src/app/components/media/MediaActionsBar.js

```diff
@@ -250,17 +250,21 @@ class MediaActionsBarComponent extends Component {
     const published = (media.dynamic_annotation_report_design && media.dynamic_annotation_report_design.data && media.dynamic_annotation_report_design.data.state === 'published');
 
     const options = [];
-    media.team.team_users.edges.forEach((teamUser) => {
-      if (teamUser.node.status === 'member' && !teamUser.node.user.is_bot) {
-        const { user } = teamUser.node;
-        options.push({ label: user.name, value: user.dbid.toString() });
-      }
-    });
+    if (media.team.team_users?.edges) {
+      media.team.team_users.edges.forEach((teamUser) => {
```

Comment:
This will work, but here's a tip: probably no need to wrap this in an IF block, you can simply use `media.team.team_users?.edges.forEach` and it should be enough!